src/components/calcite-label/calcite-label.tsx

```diff
@@ -28,7 +28,10 @@ export class CalciteLabel {
   /** specify the text alignment of the label */
   @Prop({ reflect: true }) alignment: Alignment = "start";
 
-  /** specify the status of the label and any child input / input messages */
+  /**
+   * specify the status of the label and any child input / input messages
+   * @deprecated set directly on child element instead
```

Comment:
+@benelan for editorial glance. 👀

Reply:
Looks good, thanks for changing it Yona!

Reply:
Although the prop description is kinda confusing to me but that was already there lol. The new deprecation notice is 👨‍🍳 👌